docs/sources/enterprise/reporting.md

```diff
@@ -55,16 +90,44 @@ The last saved version of the report will be sent to selected emails. You can us
 
 {{< docs-imagebox img="/img/docs/enterprise/reports_send_test_mail.png" max-width="500px" class="docs-image--no-shadow" >}}
 
+## API
```

Comment:
```suggestion
## Reports API
```

Reply:
Are there going to be more Enterprise-only APIs? If so, then we will need to properly document this and others in a separate subsection.

Reply:
Yes, that’s all but certain. Do you think we should add that already or should we wait until we have more/stable documented APIs? (I somewhat prefer the latter, otoh maybe that would mess up URLs or something like that?)

I’ll update the title.

Reply:
The thing is, we will need to remember to split this API out into a separate document when we get another one. It would mess up any links to that heading as well.

Reply:
Then I’ll split it up and link to the API doc from here 🙂